cmd/node/flags.go

```diff
@@ -246,7 +246,19 @@ var (
 	// workingDirectory defines a flag for the path for the working directory.
 	workingDirectory = cli.StringFlag{
 		Name:  "working-directory",
-		Usage: "This flag specifies the `directory` where the node will store databases, logs and statistics.",
+		Usage: "This flag specifies the `directory` where the node will store statistics.",
```

Comment:
This is now more like `statsDirectory`? Or is the `config` also loaded from here, by default?

Reply:
in the default working directory we actually have `health-records`, `stats`, and `compiledSCStorage` from what I see, so I can change the description a bit

Reply:
changed the description